pkg/hs/urbit-king/lib/Urbit/Vere/Eyre.hs

```diff
@@ -356,7 +361,11 @@ eyre env who plan isFake stderr sub = (initialEvents, runHttpServer)
   restart :: Drv -> HttpServerConf -> RIO e Serv
   restart (Drv var) conf = do
     logInfo "Restarting http server"
-    let startAct = startServ who isFake conf plan stderr sub
+    let onFatal = runRIO env $ do
+          -- XX instead maybe restart following logic under HSESetConfig below
+          stderr "You've been DDoSed. Please restart your ship."
```

Comment:
[citation needed]

Reply:
[S. T. Zargar, J. Joshi and D. Tipper, "A Survey of Defense Mechanisms Against Distributed Denial of Service (DDoS) Flooding Attacks," in IEEE Communications Surveys & Tutorials, vol. 15, no. 4, pp. 2046-2069, Fourth Quarter 2013, doi: 10.1109/SURV.2013.031413.00127.](https://sci-hub.se/https://ieeexplore.ieee.org/abstract/document/6489876)

Reply:
I don't think this error message is appropriate. If this is to be a generic error handler, what it print should be generic. If it's specifically meant to handle the ResourceLimit exception caused by leaking sockets, it shouldn't make grandiose claims about cyberwarfare.